src/main/java/racingcar/controller/CarManager.java

```diff
@@ -0,0 +1,39 @@
+package racingcar.controller;
+
+import racingcar.domain.Car;
+import racingcar.domain.CarMovement;
+import racingcar.domain.CarName;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class CarManager {
+    private List<Car> cars;
+
+    public CarManager(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void moveCarsRandomly() {
+        this.cars.stream()
+                .forEach(car -> car.move(makeRandomCarMove()));
```

Comment:
```suggestion
        this.cars.forEach(car -> car.move(makeRandomCarMove()));
```
이렇게 작성할 수도 있겠네요 ! (참고 : [Java 반복문](https://tecoble.techcourse.co.kr/post/2020-08-31-java-loop/))

Reply:
감사합니다 ㅠㅠ 덕분에 너무 좋은 메서드 하나 배워갑니다 😆